iOS/ViewController.swift

```diff
@@ -69,6 +70,12 @@ class ViewController: UIViewController, SpeakerDelegate, UIPickerViewDelegate, U
         present(alertController, animated: true, completion: nil)
     }
     
+    @IBAction func rateChanged(_ sender: UISlider) {
+        let value = roundf(100 * sender.value) / 100
```

Comment:
When to display the loudness, natural number expression is usually used.
How about divide the volume level about 10 steps?